libnavui-dropin/src/main/res/layout/mapbox_navigation_view_layout.xml

```diff
@@ -122,29 +122,23 @@
 
             </androidx.constraintlayout.widget.ConstraintLayout>
 
-            <LinearLayout
+            <FrameLayout
                 android:id="@+id/infoPanelLayout"
-                android:orientation="vertical"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:elevation="8dp"
-                android:background="@drawable/mapbox_bg_info_panel"
                 app:layout_behavior="com.google.android.material.bottomsheet.BottomSheetBehavior"
                 app:behavior_hideable="true"
                 app:behavior_fitToContents="true"
-                app:behavior_peekHeight="@dimen/mapbox_infoPanel_peekHeight">
+                app:behavior_peekHeight="@dimen/mapbox_infoPanel_peekHeight"
+                tools:background="@drawable/mapbox_bg_info_panel"
+                tools:minHeight="@dimen/mapbox_infoPanel_peekHeight">
 
-                <FrameLayout
-                    android:id="@+id/infoPanelHeader"
-                    android:layout_width="match_parent"
-                    android:layout_height="@dimen/mapbox_infoPanel_peekHeight" />
-
-                <FrameLayout
-                    android:id="@+id/infoPanelContent"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content" />
+                <!--
+                    InfoPanelBinder will inject customizable content container
+                    which will host mapbox_info_panel_layout content.
+                -->
```

Comment:
I don't think we need this comment here. It's pretty obvious with Drop-In UI architecture that views will be injected to these frame layouts later.